components/uniset/src/enum_props.rs

```diff
@@ -129,337 +144,387 @@ pub enum CanonicalCombiningClass {
+/// General descriptions of the property values are provided in the header section of the data file IndicPositionalCategory.txt.
+/// See UIndicPositionalCategory in ICU4C.
 #[derive(Clone, PartialEq, Debug)]
 pub enum IndicPositionalCategory {
-    Bottom,
-    BottomAndLeft,
-    BottomAndRight,
-    Left,
-    LeftAndRight,
-    NA,
-    Overstruck,
-    Right,
-    Top,
-    TopAndBottom,
-    TopAndBottomAndLeft,
-    TopAndBottomAndRight,
-    TopAndLeft,
-    TopAndLeftAndRight,
-    TopAndRight,
-    VisualOrderLeft,
+    Bottom = 1,
+    BottomAndLeft = 2,
+    BottomAndRight = 3,
+    Left = 4,
+    LeftAndRight = 5,
+    NA = 0,
+    Overstruck = 6,
+    Right = 7,
+    Top = 8,
+    TopAndBottom = 9,
+    TopAndBottomAndLeft = 15,
+    TopAndBottomAndRight = 10,
+    TopAndLeft = 11,
+    TopAndLeftAndRight = 12,
+    TopAndRight = 13,
+    VisualOrderLeft = 14,
 }
 
+/// A property informally defining the structural categories of syllabic components in Indic scripts.
+/// General descriptions of the property values are provided in the header section of the data file IndicSyllabicCategory.txt.
+/// See UIndicSyllabicCategory in ICU4C.
 #[derive(Clone, PartialEq, Debug)]
 pub enum IndicSyllabicCategory {
-    Avagraha,
-    Bindu,
-    BrahmiJoiningNumber,
-    CantillationMark,
-    Consonant,
-    ConsonantDead,
-    ConsonantFinal,
-    ConsonantHeadLetter,
-    ConsonantInitialPostfixed,
-    ConsonantKiller,
-    ConsonantMedial,
-    ConsonantPlaceholder,
-    ConsonantPrecedingRepha,
-    ConsonantPrefixed,
-    ConsonantSubjoined,
-    ConsonantSucceedingRepha,
-    ConsonantWithStacker,
-    GeminationMark,
-    InvisibleStacker,
-    Joiner,
-    ModifyingLetter,
-    NonJoiner,
-    Nukta,
-    Number,
-    NumberJoiner,
-    Other,
-    PureKiller,
-    RegisterShifter,
-    SyllableModifier,
-    ToneLetter,
-    ToneMark,
-    Virama,
-    Visarga,
-    Vowel,
-    VowelDependent,
-    VowelIndependent,
+    Avagraha = 1,
+    Bindu = 2,
+    BrahmiJoiningNumber = 3,
+    CantillationMark = 4,
+    Consonant = 5,
+    ConsonantDead = 6,
+    ConsonantFinal = 7,
+    ConsonantHeadLetter = 8,
+    ConsonantInitialPostfixed = 9,
+    ConsonantKiller = 10,
+    ConsonantMedial = 11,
+    ConsonantPlaceholder = 12,
+    ConsonantPrecedingRepha = 13,
+    ConsonantPrefixed = 14,
+    ConsonantSubjoined = 15,
+    ConsonantSucceedingRepha = 16,
+    ConsonantWithStacker = 17,
+    GeminationMark = 18,
+    InvisibleStacker = 19,
+    Joiner = 20,
+    ModifyingLetter = 21,
+    NonJoiner = 22,
+    Nukta = 23,
+    Number = 24,
+    NumberJoiner = 25,
+    Other = 0,
+    PureKiller = 26,
+    RegisterShifter = 27,
+    SyllableModifier = 28,
+    ToneLetter = 29,
+    ToneMark = 30,
+    Virama = 31,
+    Visarga = 32,
+    Vowel = 33,
+    VowelDependent = 34,
+    VowelIndependent = 35,
 }
 
+// TODO: suggest update (to include more script categories) for description in UAX 44.
```

Comment:
script categories?

Reply:
I don't remember what part of the document I was thinking of, but "script categories" doesn't make sense so doesn't help me remember. At best, it was a minor change, but ok to ignore either way.